src/contexts/BackendContext.jsx

```diff
@@ -23,31 +23,31 @@ const BackendProvider = ({ children }) => {
   // or if the token is invalid we can send a 401 response
   // no need to do anything on the backend
 
-//   backend.interceptors.request.use(
-//     async config => {
-//       if (currentUser) {
-//         // get token from firebase if there's a current user
-//         const token = await currentUser?.getIdToken();
-//         // eslint-disable-next-line
-//         config.headers.authorization = `Bearer ${token}`;
-//       }
-//       return config;
-//     },
-//     error => Promise.reject(error),
-//   );
-
-//   backend.interceptors.response.use(
-//     response => response,
-//     error => {
-//       // import 404 redirection from utils.jsx here
-//       if (error?.response?.status === 404) {
-//         navigate('*');
-//       }
-//       return Promise.reject(error.response);
-//     },
-//   );
+  //   backend.interceptors.request.use(
+  //     async config => {
+  //       if (currentUser) {
+  //         // get token from firebase if there's a current user
+  //         const token = await currentUser?.getIdToken();
+  //         // eslint-disable-next-line
+  //         config.headers.authorization = `Bearer ${token}`;
+  //       }
+  //       return config;
+  //     },
+  //     error => Promise.reject(error),
+  //   );
+
+  //   backend.interceptors.response.use(
+  //     response => response,
+  //     error => {
+  //       // import 404 redirection from utils.jsx here
+  //       if (error?.response?.status === 404) {
+  //         navigate('*');
+  //       }
+  //       return Promise.reject(error.response);
+  //     },
+  //   );
 
   return <BackendContext.Provider value={{ backend }}>{children}</BackendContext.Provider>;
 };
 
-export { BackendProvider, useBackend };
\ No newline at end of file
+export { BackendProvider, useBackend };
```

Comment:
⚠️ **[eslint]** <react-refresh/only-export-components> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.